inst/templates/circleci-config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: {{{ image }}}
+    steps:
+      - restore_cache:
+          keys:
+            - cache
+      - checkout
+      - run:
+          name: Install package dependencies
+          command: R -e "devtools::install_deps(dependencies = TRUE)"
```

Comment:
Probably best to use the remotes packages for this, something like

```r
R -e 'if (!require("remotes")) install.packages("remotes")' -e 'remotes::install_deps(dependencies = TRUE)'
```

We might even just want to unconditionally call `install.packages("remotes")` if for no other reason that to ensure it is installed and is the latest version from CRAN.

Reply:
To ensure it was always up-to-date, I think we'd need to unconditionally install it. The rocker image verse already has remotes/devtools installed.